osu.Game/Localisation/BeatmapOverlayStrings.cs

```diff
@@ -0,0 +1,34 @@
+﻿// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the MIT Licence.
+// See the LICENCE file in the repository root for full licence text.
+
+using osu.Framework.Localisation;
+
+namespace osu.Game.Localisation
+{
+    public static class BeatmapOverlayStrings
+    {
+        private const string prefix = @"osu.Game.Resources.Localisation.BeatmapOverlayStrings";
+
+        /// <summary>
+        /// "User content disclaimer"
+        /// </summary>
+        public static LocalisableString UserContentDisclaimer => new TranslatableString(getKey(@"user_content_disclaimer"), @"User content disclaimer");
+
+        /// <summary>
+        /// "By turning off the &quot;featured artist&quot; filter, all user uploaded content will be displayed.
+        ///
+        /// This includes content which may not be correctly licensed for use and as such may not be safe for streaming, sharing, or consumption."
+        /// </summary>
+        public static LocalisableString ByTurningOffTheFeatured => new TranslatableString(getKey(@"by_turning_off_the_featured"),
+            @"By turning off the ""featured artist"" filter, all user uploaded content will be displayed.
+
+This includes content which may not be correctly licensed for use and as such may not be safe for streaming, sharing, or consumption.");
```

Comment:
I think this disclaimer has issues. 

Implying that not featured content is not safe for streaming or sharing also implies that songs from featured artists are. Maybe that's fine, but I've heard not all featured artists songs are good for streaming. Also using the word consumption is very weird. It's almost like admitting people use osu! for non-rhythm game related things. I think the disclaimer would read better as "This includes content that may not be correctly licensed for osu! usage. Browse at your own risk."

Reply:
Sounds fine to me, much simpler and to the point. I was trying to be extra verbose but agree it may not cover 100% of cases.